components/UserProvider.js

```diff
@@ -116,13 +116,11 @@ class UserProvider extends React.Component {
         return null;
       }
 
-      // Store the error
-      this.setState({ loadingLoggedInUser: false, errorLoggedInUser: error.message });
```

Comment:
Why remove this?

Reply:
This is just pushed to the bottom because it is still loading if we are proceeding to the 2FA while loop